app/views/kaui/accounts/_form.html.erb

```diff
@@ -9,9 +9,14 @@
       </div>
     </div>
     <div class="form-group">
-      <%= f.label :first_name_length, 'First name length', :class => 'col-sm-3 control-label' %>
-      <div class="col-sm-9">
-        <%= f.text_field :first_name_length, :class => 'form-control' %>
+      <%= f.label :first_name_length, :class => 'col-sm-3 control-label' do %>
+          First name length<%= link_to '*', 'https://www.w3.org/International/questions/qa-personal-names',target: '_blank', rel: 'nofollow' %>
+      <% end %>
+          <div class="col-sm-9">
+        <%= f.text_field :first_name_length,
+                         :pattern => '^[0-9]*$',
```

Comment:
👍 

Could we also populate a default? We could split the name field to the first blank space and use the left hand side: it would be 10 for `María-Jose Carreño Quiñones` for example. This is wrong of course for right-to-left languages (e.g. 毛泽东) but it's probably acceptable. Let's also leave it blank (unfilled, not zero) in case there are no spaces.